contracts/BugBounty.sol

```diff
@@ -2,15 +2,48 @@ pragma solidity ^0.4.4;
 
 import 'zeppelin-solidity/contracts/ownership/Ownable.sol';
 
+/// @title BugBounty - A basic bug bounty program
```

Comment:
we should do multiline comment style, and otherwise match what OpenZeppelin is doing, for example https://github.com/OpenZeppelin/zeppelin-solidity/blob/master/contracts/token/MintableToken.sol#L9 . I think it will make it easier to run document generation once we're ready to set that up